espnet/nets/pytorch_backend/e2e_tts_transformer.py

```diff
@@ -13,6 +13,7 @@
 
 from espnet.nets.pytorch_backend.e2e_asr_transformer import subsequent_mask
 from espnet.nets.pytorch_backend.e2e_tts_tacotron2 import GuidedAttentionLoss
+from espnet.nets.pytorch_backend.e2e_tts_tacotron2 import Tacotron2Loss as TransformerLoss
```

Comment:
Oh :)

We could consider naming it a more clear name that denotes what it actually does. Not an issue of the PR though.